examples/asa/scripts/transfer/gold-delegated-lsig.js

```diff
@@ -54,7 +62,11 @@ async function run(runtimeEnv, deployer) {
 
 	// Transaction FAIL - rejected by lsig because amount is not <= 100
 	txnParam.amountMicroAlgos = 580;
-	await tryExecuteTx(deployer, txnParam);
+	try {
```

Comment:
will the console.error appear in GitHub CI?

Reply:
I think so, it should

Reply:
I mean if it will make the CI fail

Reply:
yes it will, the logs will be in red color and few other things
but since we are using the throw error, it is almost same